src/App.js

```diff
@@ -11,6 +12,8 @@ import {FirstReactComponent} from './FirstReactComponent.jsx';
       <ChildrenWithTitle title="I am title">
         I am children
       </ChildrenWithTitle>
+      <ConditionalRenderingComponent isBlue = {true} />
+      <ConditionalRenderingComponent isBlue = {false} />
```

Comment:
Just a reminder, that when do not provide the property it resolves into a undefined which is a falsy statement, there fore you don't actually need to write the `isBlue={false}` :)